android/app/src/main/res/layout/place_page_website_menu.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:id="@+id/ll__place_website_menu"
+  style="@style/PlacePageItemFrame"
+  android:tag="website"
+  tools:background="#20FF0000"
+  tools:visibility="visible">
+
+  <ImageView
+    android:id="@+id/iv__place_website_menu"
+    style="@style/PlacePageMetadataIcon"
+    app:srcCompat="@drawable/ic_github"
```

Comment:
Why Github icon?

Reply:
For now it's a test. I will change it. But, how icons are designed for organic maps ?

Reply:
On UI, we use icons proposed by Google and Apple.
WDYT to you same icons has website tag or [menu icon](https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Amenu_book%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4024)

Reply:
This [icon](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:restaurant_menu:FILL@0;wght@400;GRAD@0;opsz@24&icon.size=24&icon.color=%23e8eaed&icon.query=menu) look good